Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,13 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+a) O(n)
+
+b) O(n^4)
```

Comment:
The fourth loop is constant since it iterates 10 times no matter what. Overall O(n^3)